msteams-platform/tabs/how-to/authentication/auth-aad-sso.md

```diff
@@ -19,19 +16,21 @@ The following diagram shows how the SSO process works:
 
 1. In the tab, JavaScript calls getAuthToken(). This tells the Teams application to obtain an authentication token to the tab application.
 2. If this is the first time the current user has used your tab application, they will be prompted to consent (if consent is required) or asked to handle step-up authentication (such as two-factor authentication).
-3. The Microsoft Teams application requests the tab application token from the Azure AD v1.0 endpoint for the current user.
+3. The Microsoft Teams application requests the tab application token from the Azure AD endpoint for the current user.
 4. Azure AD sends the tab application token to the Teams application.
 5. The Microsoft Teams application sends the tab application token to the tab as part of the result object returned by the getAuthToken() call.
 6. JavaScript in the tab application can parse the token and extract the information it needs, such as the user's email address.
-    * Note: this token is only valid for consenting to a limited set of user-level APIs (ex: email, profile, etc)  and not for further Graph scopes (such as Mail.Read). See our section at the end of this document for suggested workarounds if you require additional Graph scopes.
+    * Note: this token is only valid for consenting to a limited set of user-level APIs (ie: email, profile, offline_access and openid)  and not for further Graph scopes (such as User.Read or Mail.Read). See our section at the end of this document for suggested workarounds if you require additional Graph scopes.
+
+The SSO APIs will also work in [Task Modules](~/task-modules-and-cards/what-are-task-modules.md) that embed web content.
 
 ## Develop an SSO Microsoft Teams tab
 
 This section describes the tasks involved in creating an Microsoft Teams tab that use SSO. These tasks are described here in a language- and framework-agnostic way.
 
 ### 1. Create your AAD application in Azure
 
-Register you application at the registration portal for the Azure AD v1.0 endpoint. This is a 5–10 minute process that includes the following tasks:
+Register you application at the registration portal for the Azure AD endpoint. This is a 5–10 minute process that includes the following tasks:
 
 * Getting your AAD application ID
```

Comment:
Consider changing the bullet points to the same tense. i.e
Getting....
Specifying....
Granting....